CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,3 @@
+Add:  POST /v2/entities - entity creation under version 2 (Issue #970)
+Add:  Version 2 only supports JSON so we can now distinguish between STRINGS, NUMBERS (all treated as floats for now),
```

Comment:
"Version 2" could be missleading... I would suggest "API version 2" to make clear that we refer to the API.

Moreover, I would use "for the VALUE for ContextAttributes and Metadata". Compounds is actually part of the ContextAttribute from a user perspective (changelog is mostly for users).

Regarding issue number, not every entry in CNR needs an issue number. I think is ok the way it is now.